### ISSUE TITLE
[Config] improve Config

### ISSUE THREAD

**Issue by ddmler** [ovr/phpsa#234]
- [ ] disabling/enabling notices by levels like CHECK_SAFE/CHECK_BETA/…
- [ ] improving documentation regarding analyzers configuration
- [ ] improving notices generated by analyzers to include a better description and a link to a page (in the wiki probably)
- [ ] check minimum php version for analyzers (for example shortarray needs 5.4 or something)

**Comment by ddmler** [comment]
minimum php versions still needed:
ArgumentUnpacking: 5.6
Deprecated Function /INI: multiple different versions in the code. (split those up into multiple analyzers?? if yes create new directory for deprecated analyzers))

**Comment by ovr** [comment]
@ddmler Looks like > check minimum php version for analyzers
is implemented now

I change from [ ] to [X] for it

**Comment by ddmler** [comment]
@ovr yeah missing is deprecated function/ini analyzer with minimum php version.

**Comment by Mte90** [comment]
I think that also a way to exclude files be very useful.

**Comment by Kilgaloon** [comment]
@Mte90 there is [feature] regarding that #344
